TransformationSystem/Agent/TaskManagerAgentBase.py

```diff
@@ -66,6 +69,10 @@ def initialize( self ):
         - list of transformation types to be looked (self.transType)
     """
 
+    self.moduloNumber = self.am_getOption('moduloNumber',0)
+    self.moduloResidual = self.am_getOption('moduloResidual',-1)
+    self.log.info('Split agents modulo %d, this agend resdiual %d' % (self.moduloNumber, self.moduloResidual))
```

Comment:
If self.moduloNumber == 0, no modulo logic is requested ( the default ). In this case no need for this log message. The default behavior must stay intact. When we will find a better solution to this problem, we will just drop this modulo logic.